src/main/java/com/example/speedwayapi/RaceService.java

```diff
@@ -0,0 +1,40 @@
+package com.example.speedwayapi;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class RaceService {
+    private final RaceRepository raceRepository;
+
+    @Autowired
```

Comment:
Not sure this is needed as you are doing the **Dependency Injection** (DI) already, but maybe this didn't work without it? It compiles and passes tests it should be fine.